### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by shloch**

### CODE REVIEW

File: enumerables.rb [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+    # 7 - Create #my_count
+    def my_count(arg=nil)
+      if !(arg.nil?)
+        i = 0
+        self.my_each do |item|
+            i += 1 if item == arg
+        end
+        return i
+      elsif block_given?
+        return self.my_select {|item| yield(item)}.length
```
Comment (inhaq) [inline enumerables.rb]: Don't use `my_select` or `length`

**Review by inhaq — CHANGES_REQUESTED** [review]
### Well done with Enumerables 🏅
You can remove comments/puts etc and use linter to make your code more readable. There's also minor change required, kindly fix it. Good work with the functionality.
Once fixed, kindly resubmit for review.

File: enumerables.rb (hunk 1 above)
Comment (shloch) [reply enumerables.rb]: thanks bro for the review..i have resubmitted a proposed correction

**Review by inhaq — APPROVED** [review]
### Now it's clean 🏆
Good work. It's 👍🏻 for this project. I would like to suggest to add `slicker-ci` to the projects to make code much clearer in the future. You can always use `rubocop` locally. 
Good luck with the next project 🎉

**Pull request merged**